code/modules/clothing/under/miscellaneous.dm

```diff
@@ -572,3 +572,19 @@
 	item_state = "telvis"
 	item_color = "telvis"
 	can_adjust = 0
+
+/obj/item/clothing/under/zootsuit
```

Comment:
Is this how I reply?

If so:

Nah, I think I might have added a line there or something tho.